react/components/ProductDescription/index.js

```diff
@@ -12,35 +12,42 @@ import './global.css'
  */
 class ProductDescription extends Component {
   render() {
-    const { specifications, skuName, description } = this.props
+    const { specifications, skuName, description, rowClasses } = this.props
+    let { classes } = this.props
 
     if (!description || !specifications) {
       return null
     }
 
+    classes = {
+      ...ProductDescription.defaultProps.classes,
+      ...classes
+    }
+
     return (
-      <div className="vtex-product-description ma2">
-        <div className="f4 b ttu mb3">
-          <FormattedMessage id="product-description.title" />
+      <div className={classes.root}>
+        <div className={classes.description.container}>
```

Comment:
Destructure classes to avoid repetition and improve readability.

Reply:
I agree.

Reply:
There is a problem related to that I didn't destructure for the same reason that a didn't in `ProductName` component. Here is [the reason](https://github.com/vtex-apps/store-components/pull/183) that I discussed with @luciannojunior

Reply:
One option is renaming the props:

`const { root: rootClass } = classes`

Reply:
> One option is renaming the props:
> 
> `const { root: rootClass } = classes`

I like this idea

Reply:
But even then it seems odd, because you're changing `classes.root` for `rootClass`

Reply:
I think it describes better the variable and in cases where you must use a value that is more than one level depp in the object it still reduces duplication